ocaml/tests/test_pool_update.ml

```diff
@@ -41,19 +41,20 @@ let test_assert_space_available () =
 
 let test_download_restriction () =
   Xapi_globs.host_update_dir := ".";
-  let assert_no_dots s =
+  let assert_no_dots (_, s) =
     Alcotest.(check bool)
       "test_download_restriction: path must not contain ."
       true
       (String.index_opt s '.' = None)
   in
+  let uuid = Helpers.get_localhost_uuid () in
```

Comment:
You are right, Rob. Probably `get_localhost_uuid ` returns blank and it still works below. But I found another place in the file that use it too, so I refine it with a `fake` uuid there to avoid confusion. Thanks.
https://github.com/xapi-project/xen-api/blame/master/ocaml/tests/test_pool_update.ml#L28